core/state-machine/src/testing.rs

```diff
@@ -187,7 +241,7 @@ mod tests {
 		ext.set_storage(b"doe".to_vec(), b"reindeer".to_vec());
 		ext.set_storage(b"dog".to_vec(), b"puppy".to_vec());
 		ext.set_storage(b"dogglesworth".to_vec(), b"cat".to_vec());
-		const ROOT: [u8; 32] = hex!("0b33ed94e74e0f8e92a55923bece1ed02d16cf424e124613ddebc53ac3eeeabe");
+		const ROOT: [u8; 32] = hex!("cc65c26c37ebd4abcdeb3f1ecd727527051620779a2f6c809bac0f8a87dbb816");
```

Comment:
is this actually a test failure ? I shouldn't be required to change the root no ? @cheme

Reply:
I would expect it to be the same, so there should be a change in content between both impl.

Reply:
https://github.com/paritytech/substrate/pull/2502#discussion_r282503434